core/src/main/kotlin/io/fritz2/dom/html/html.kt

```diff
@@ -9,8 +9,20 @@ import org.w3c.dom.Element
 @FlowPreview
 fun <E : Element> html(content: HtmlElements.() -> Tag<E>) =
     content(object : HtmlElements {
+
+        var alreadyRegistered: Boolean = false
+
         override fun <X : Element, T : Tag<X>> register(element: T, content: (T) -> Unit): T {
-            content(element)
-            return element
+            if (!alreadyRegistered) {
+                content(element)
+                alreadyRegistered = true
+                return element
+            } else {
+                throw MultipleRootElementsException(
+                    "Don't use multiple root elements/tags in the html{...} context of your component! That's why rendering is not working."
```

Comment:
Suggestion:
You can have only one root-tag per html-context.

Reply:
Ok, I have changed it.